### ISSUE TITLE
Kamailio doesn't start with db_mongodb and usrloc.

### ISSUE THREAD

**Issue by tiglat** [kamailio/kamailio#1507]
### Description
Kamailio doesn't start with db_mongodb and usrloc.

Configuration:
kamailio 4.4.7 (tried to use db_mongodb module from 5.1.2)
usrloc is configured to use db_mode=1 and db_insert_null=1.
MongoDB 3.0.2 and 3.6.3
mongo-c-driver was tested 1.9.2 and 1.9.4.
location table contains 300 records.

#### Reproduction

1. Configure db_mode=1 (use internal cache). 
2. Make 300 successful registrations to have 300 records inside mongodb. 
3. Restart Kamailio

#### Debugging Data

db_mongodb_base.c 
function db_mongodb_convert_bson()
line 614

#### Log Messages

```

64) DEBUG: usrloc [udomain.c:469]: preload_udomain(): loading records - cycle [1]
1(13464) CRITICAL: usrloc [udomain.c:476]: preload_udomain(): empty username record in table location...skipping
1(13464) CRITICAL: usrloc [udomain.c:476]: preload_udomain(): empty username record in table location...skipping
1(13464) DEBUG: <core> [socket_info.c:564]: grep_sock_info(): checking if host==us: 1==15 && [s] == [192.168.158.178]
1(13464) DEBUG: <core> [socket_info.c:583]: grep_sock_info(): checking advertise if host==us: 1==15 && [s] == [192.168.158.178]
1(13464) DEBUG: <core> [socket_info.c:564]: grep_sock_info(): checking if host==us: 1==15 && [s] == [192.168.158.178]
1(13464) DEBUG: <core> [socket_info.c:583]: grep_sock_info(): checking advertise if host==us: 1==15 && [s] == [192.168.158.178]
1(13464) DEBUG: <core> [socket_info.c:564]: grep_sock_info(): checking if host==us: 1==15 && [s] == [192.168.158.178]
1(13464) DEBUG: <core> [socket_info.c:583]: grep_sock_info(): checking advertise if host==us: 1==15 && [s] == [192.168.158.178]
1(13464) DEBUG: <core> [socket_info.c:564]: grep_sock_info(): checking if host==us: 1==15 && [s] == [192.168.158.178]
1(13464) DEBUG: <core> [socket_info.c:583]: grep_sock_info(): checking advertise if host==us: 1==15 && [s] == [192.168.158.178]
1(13464) DEBUG: usrloc [udomain.c:313]: dbrow2info(): non-local socket <s>...ignoring
1(13464) DEBUG: <core> [socket_info.c:564]: grep_sock_info(): checking if host==us: 2==15 && [gs] == [192.168.158.178]
1(13464) DEBUG: <core> [socket_info.c:583]: grep_sock_info(): checking advertise if host==us: 2==15 && [gs] == [192.168.158.178]
1(13464) DEBUG: <core> [socket_info.c:564]: grep_sock_info(): checking if host==us: 2==15 && [gs] == [192.168.158.178]
1(13464) DEBUG: <core> [socket_info.c:583]: grep_sock_info(): checking advertise if host==us: 2==15 && [gs] == [192.168.158.178]
1(13464) DEBUG: <core> [socket_info.c:564]: grep_sock_info(): checking if host==us: 2==15 && [gs] == [192.168.158.178]
1(13464) DEBUG: <core> [socket_info.c:583]: grep_sock_info(): checking advertise if host==us: 2==15 && [gs] == [192.168.158.178]
1(13464) DEBUG: <core> [socket_info.c:564]: grep_sock_info(): checking if host==us: 2==15 && [gs] == [192.168.158.178]
1(13464) DEBUG: <core> [socket_info.c:583]: grep_sock_info(): checking advertise if host==us: 2==15 && [gs] == [192.168.158.178]
1(13464) DEBUG: usrloc [udomain.c:313]: dbrow2info(): non-local socket <gs>...ignoring
1(13464) DEBUG: <core> [socket_info.c:564]: grep_sock_info(): checking if host==us: 1==15 && [s] == [192.168.158.178]
1(13464) DEBUG: <core> [socket_info.c:583]: grep_sock_info(): checking advertise if host==us: 1==15 && [s] == [192.168.158.178]
1(13464) DEBUG: <core> [socket_info.c:564]: grep_sock_info(): checking if host==us: 1==15 && [s] == [192.168.158.178]
1(13464) DEBUG: <core> [socket_info.c:583]: grep_sock_info(): checking advertise if host==us: 1==15 && [s] == [192.168.158.178]
1(13464) DEBUG: <core> [socket_info.c:564]: grep_sock_info(): checking if host==us: 1==15 && [s] == [192.168.158.178]
1(13464) DEBUG: <core> [socket_info.c:583]: grep_sock_info(): checking advertise if host==us: 1==15 && [s] == [192.168.158.178]
1(13464) DEBUG: <core> [socket_info.c:564]: grep_sock_info(): checking if host==us: 1==15 && [s] == [192.168.158.178]
1(13464) DEBUG: <core> [socket_info.c:583]: grep_sock_info(): checking advertise if host==us: 1==15 && [s] == [192.168.158.178]
1(13464) DEBUG: usrloc [udomain.c:313]: dbrow2info(): non-local socket <s>...ignoring
1(13464) CRITICAL: usrloc [udomain.c:476]: preload_udomain(): empty username record in table location...skipping
1(13464) CRITICAL: usrloc [udomain.c:476]: preload_udomain(): empty username record in table location...skipping
1(13464) CRITICAL: usrloc [udomain.c:476]: preload_udomain(): empty username record in table location...skipping
1(13464) CRITICAL: usrloc [udomain.c:476]: preload_udomain(): empty username record in table location...skipping
1(13464) CRITICAL: usrloc [udomain.c:476]: preload_udomain(): empty username record in table location...skipping
1(13464) CRITICAL: usrloc [udomain.c:476]: preload_udomain(): empty username record in table location...skipping
1(13464) CRITICAL: usrloc [udomain.c:256]: dbrow2info(): bad callid
1(13464) ERROR: usrloc [udomain.c:484]: preload_udomain(): skipping record for kgate.zz in table location
1(13464) DEBUG: <core> [socket_info.c:564]: grep_sock_info(): checking if host==us: 1==15 && [t] == [192.168.158.178]
1(13464) DEBUG: <core> [socket_info.c:583]: grep_sock_info(): checking advertise if host==us: 1==15 && [t] == [192.168.158.178]
1(13464) DEBUG: <core> [socket_info.c:564]: grep_sock_info(): checking if host==us: 1==15 && [t] == [192.168.158.178]
1(13464) DEBUG: <core> [socket_info.c:583]: grep_sock_info(): checking advertise if host==us: 1==15 && [t] == [192.168.158.178]
1(13464) DEBUG: <core> [socket_info.c:564]: grep_sock_info(): checking if host==us: 1==15 && [t] == [192.168.158.178]
1(13464) DEBUG: <core> [socket_info.c:583]: grep_sock_info(): checking advertise if host==us: 1==15 && [t] == [192.168.158.178]
1(13464) DEBUG: <core> [socket_info.c:564]: grep_sock_info(): checking if host==us: 1==15 && [t] == [192.168.158.178]
1(13464) DEBUG: <core> [socket_info.c:583]: grep_sock_info(): checking advertise if host==us: 1==15 && [t] == [192.168.158.178]
1(13464) DEBUG: usrloc [udomain.c:313]: dbrow2info(): non-local socket <t>...ignoring
1(13464) CRITICAL: usrloc [udomain.c:476]: preload_udomain(): empty username record in table location...skipping
1(13464) CRITICAL: usrloc [udomain.c:476]: preload_udomain(): empty username record in table location...skipping
1(13464) DEBUG: <core> [socket_info.c:564]: grep_sock_info(): checking if host==us: 8==15 && [received] == [192.168.158.178]
1(13464) DEBUG: <core> [socket_info.c:583]: grep_sock_info(): checking advertise if host==us: 8==15 && [received] == [192.168.158.178]
1(13464) DEBUG: <core> [socket_info.c:564]: grep_sock_info(): checking if host==us: 8==15 && [received] == [192.168.158.178]
1(13464) DEBUG: <core> [socket_info.c:583]: grep_sock_info(): checking advertise if host==us: 8==15 && [received] == [192.168.158.178]
1(13464) DEBUG: <core> [socket_info.c:564]: grep_sock_info(): checking if host==us: 8==15 && [received] == [192.168.158.178]
1(13464) DEBUG: <core> [socket_info.c:583]: grep_sock_info(): checking advertise if host==us: 8==15 && [received] == [192.168.158.178]
1(13464) DEBUG: <core> [socket_info.c:564]: grep_sock_info(): checking if host==us: 8==15 && [received] == [192.168.158.178]
1(13464) DEBUG: <core> [socket_info.c:583]: grep_sock_info(): checking advertise if host==us: 8==15 && [received] == [192.168.158.178]
1(13464) DEBUG: usrloc [udomain.c:313]: dbrow2info(): non-local socket <received>...ignoring
1(13464) CRITICAL: usrloc [udomain.c:476]: preload_udomain(): empty username record in table location...skipping
1(13464) DEBUG: <core> [socket_info.c:564]: grep_sock_info(): checking if host==us: 8==15 && [received] == [192.168.158.178]
1(13464) DEBUG: <core> [socket_info.c:583]: grep_sock_info(): checking advertise if host==us: 8==15 && [received] == [192.168.158.178]
1(13464) DEBUG: <core> [socket_info.c:564]: grep_sock_info(): checking if host==us: 8==15 && [received] == [192.168.158.178]
1(13464) DEBUG: <core> [socket_info.c:583]: grep_sock_info(): checking advertise if host==us: 8==15 && [received] == [192.168.158.178]
1(13464) DEBUG: <core> [socket_info.c:564]: grep_sock_info(): checking if host==us: 8==15 && [received] == [192.168.158.178]
1(13464) DEBUG: <core> [socket_info.c:583]: grep_sock_info(): checking advertise if host==us: 8==15 && [received] == [192.168.158.178]
1(13464) DEBUG: <core> [socket_info.c:564]: grep_sock_info(): checking if host==us: 8==15 && [received] == [192.168.158.178]
1(13464) DEBUG: <core> [socket_info.c:583]: grep_sock_info(): checking advertise if host==us: 8==15 && [received] == [192.168.158.178]
1(13464) DEBUG: usrloc [udomain.c:313]: dbrow2info(): non-local socket <received>...ignoring
1(13464) CRITICAL: usrloc [udomain.c:476]: preload_udomain(): empty username record in table location...skipping
1(13464) CRITICAL: usrloc [udomain.c:256]: dbrow2info(): bad callid
1(13464) ERROR: usrloc [udomain.c:484]: preload_udomain(): skipping record for Þñª(%Brusername in table location
1(13464) CRITICAL: usrloc [udomain.c:256]: dbrow2info(): bad callid
1(13464) ERROR: usrloc [udomain.c:484]: preload_udomain(): skipping record for ª(%Bsusername in table location
1(13464) CRITICAL: usrloc [udomain.c:231]: dbrow2info(): bad contact
1(13464) ERROR: usrloc [udomain.c:484]: preload_udomain(): skipping record for %Btusername in table location
1(13464) DEBUG: <core> [socket_info.c:564]: grep_sock_info(): checking if host==us: 3==15 && [ket] == [192.168.158.178]
1(13464) DEBUG: <core> [socket_info.c:583]: grep_sock_info(): checking advertise if host==us: 3==15 && [ket] == [192.168.158.178]
1(13464) DEBUG: <core> [socket_info.c:564]: grep_sock_info(): checking if host==us: 3==15 && [ket] == [192.168.158.178]
1(13464) DEBUG: <core> [socket_info.c:583]: grep_sock_info(): checking advertise if host==us: 3==15 && [ket] == [192.168.158.178]
1(13464) DEBUG: <core> [socket_info.c:564]: grep_sock_info(): checking if host==us: 3==15 && [ket] == [192.168.158.178]
1(13464) DEBUG: <core> [socket_info.c:583]: grep_sock_info(): checking advertise if host==us: 3==15 && [ket] == [192.168.158.178]
1(13464) DEBUG: <core> [socket_info.c:564]: grep_sock_info(): checking if host==us: 3==15 && [ket] == [192.168.158.178]
1(13464) DEBUG: <core> [socket_info.c:583]: grep_sock_info(): checking advertise if host==us: 3==15 && [ket] == [192.168.158.178]
1(13464) DEBUG: usrloc [udomain.c:313]: dbrow2info(): non-local socket <ket>...ignoring
1(13464) CRITICAL: usrloc [udomain.c:492]: preload_udomain(): empty domain record for user ame...skipping
1(13464) CRITICAL: usrloc [udomain.c:476]: preload_udomain(): empty username record in table location...skipping
1(13464) CRITICAL: usrloc [udomain.c:476]: preload_udomain(): empty username record in table location...skipping
1(13464) DEBUG: <core> [socket_info.c:564]: grep_sock_info(): checking if host==us: 15==15 && [192.168.158.178] == [192.168.158.178]
1(13464) DEBUG: <core> [socket_info.c:567]: grep_sock_info(): checking if port 5060 (advertise 5060) matches port 16060
1(13464) DEBUG: <core> [socket_info.c:564]: grep_sock_info(): checking if host==us: 15==15 && [192.168.158.178] == [192.168.158.178]
1(13464) DEBUG: <core> [socket_info.c:567]: grep_sock_info(): checking if port 16060 (advertise 16060) matches port 16060
1(13464) WARNING: usrloc [../../ut.h:717]: shm_str_dup(): shm_str_dup fallback; dup called for src->s == NULL or src->len < 0
1(13464) WARNING: usrloc [../../ut.h:731]: shm_str_dup(): shm_str_dup fallback; skip memcpy for src->s == NULL
1(13464) DEBUG: <core> [socket_info.c:564]: grep_sock_info(): checking if host==us: 12==15 && [.168.158.178] == [192.168.158.178]
1(13464) DEBUG: <core> [socket_info.c:567]: grep_sock_info(): checking if port 5060 (advertise 5060) matches port 16060
1(13464) DEBUG: <core> [socket_info.c:564]: grep_sock_info(): checking if host==us: 12==15 && [.168.158.178] == [192.168.158.178]
1(13464) DEBUG: <core> [socket_info.c:567]: grep_sock_info(): checking if port 16060 (advertise 16060) matches port 16060
1(13464) DEBUG: <core> [socket_info.c:583]: grep_sock_info(): checking advertise if host==us: 12==15 && [.168.158.178] == [192.168.158.178]
1(13464) DEBUG: <core> [socket_info.c:564]: grep_sock_info(): checking if host==us: 12==15 && [.168.158.178] == [192.168.158.178]
1(13464) DEBUG: <core> [socket_info.c:567]: grep_sock_info(): checking if port 5060 (advertise 5060) matches port 16060
1(13464) DEBUG: <core> [socket_info.c:564]: grep_sock_info(): checking if host==us: 12==15 && [.168.158.178] == [192.168.158.178]
1(13464) DEBUG: <core> [socket_info.c:567]: grep_sock_info(): checking if port 16060 (advertise 16060) matches port 16060
1(13464) DEBUG: <core> [socket_info.c:583]: grep_sock_info(): checking advertise if host==us: 12==15 && [.168.158.178] == [192.168.158.178]
1(13464) DEBUG: usrloc [udomain.c:313]: dbrow2info(): non-local socket <.168.158.178:16060>...ignoring
1(13464) DEBUG: <core> [socket_info.c:564]: grep_sock_info(): checking if host==us: 15==15 && [192.168.158.178] == [192.168.158.178]
1(13464) DEBUG: <core> [socket_info.c:567]: grep_sock_info(): checking if port 5060 (advertise 5060) matches port 16060
1(13464) DEBUG: <core> [socket_info.c:564]: grep_sock_info(): checking if host==us: 15==15 && [192.168.158.178] == [192.168.158.178]
1(13464) DEBUG: <core> [socket_info.c:567]: grep_sock_info(): checking if port 16060 (advertise 16060) matches port 16060
1(13464) WARNING: usrloc [../../ut.h:717]: shm_str_dup(): shm_str_dup fallback; dup called for src->s == NULL or src->len < 0
1(13464) WARNING: usrloc [../../ut.h:731]: shm_str_dup(): shm_str_dup fallback; skip memcpy for src->s == NULL
1(13464) DEBUG: <core> [socket_info.c:564]: grep_sock_info(): checking if host==us: 12==15 && [.168.158.178] == [192.168.158.178]
1(13464) DEBUG: <core> [socket_info.c:567]: grep_sock_info(): checking if port 5060 (advertise 5060) matches port 16060
1(13464) DEBUG: <core> [socket_info.c:564]: grep_sock_info(): checking if host==us: 12==15 && [.168.158.178] == [192.168.158.178]
1(13464) DEBUG: <core> [socket_info.c:567]: grep_sock_info(): checking if port 16060 (advertise 16060) matches port 16060
1(13464) DEBUG: <core> [socket_info.c:583]: grep_sock_info(): checking advertise if host==us: 12==15 && [.168.158.178] == [192.168.158.178]
1(13464) DEBUG: <core> [socket_info.c:564]: grep_sock_info(): checking if host==us: 12==15 && [.168.158.178] == [192.168.158.178]
1(13464) DEBUG: <core> [socket_info.c:567]: grep_sock_info(): checking if port 5060 (advertise 5060) matches port 16060
1(13464) DEBUG: <core> [socket_info.c:564]: grep_sock_info(): checking if host==us: 12==15 && [.168.158.178] == [192.168.158.178]
1(13464) DEBUG: <core> [socket_info.c:567]: grep_sock_info(): checking if port 16060 (advertise 16060) matches port 16060
1(13464) DEBUG: <core> [socket_info.c:583]: grep_sock_info(): checking advertise if host==us: 12==15 && [.168.158.178] == [192.168.158.178]
1(13464) DEBUG: usrloc [udomain.c:313]: dbrow2info(): non-local socket <.168.158.178:16060>...ignoring
1(13464) DEBUG: <core> [socket_info.c:564]: grep_sock_info(): checking if host==us: 9==15 && [8.158.178] == [192.168.158.178]
1(13464) DEBUG: <core> [socket_info.c:567]: grep_sock_info(): checking if port 5060 (advertise 5060) matches port 16060
1(13464) DEBUG: <core> [socket_info.c:564]: grep_sock_info(): checking if host==us: 9==15 && [8.158.178] == [192.168.158.178]
1(13464) DEBUG: <core> [socket_info.c:567]: grep_sock_info(): checking if port 16060 (advertise 16060) matches port 16060
1(13464) DEBUG: <core> [socket_info.c:583]: grep_sock_info(): checking advertise if host==us: 9==15 && [8.158.178] == [192.168.158.178]
1(13464) DEBUG: <core> [socket_info.c:564]: grep_sock_info(): checking if host==us: 9==15 && [8.158.178] == [192.168.158.178]
1(13464) DEBUG: <core> [socket_info.c:567]: grep_sock_info(): checking if port 5060 (advertise 5060) matches port 16060
1(13464) DEBUG: <core> [socket_info.c:564]: grep_sock_info(): checking if host==us: 9==15 && [8.158.178] == [192.168.158.178]
1(13464) DEBUG: <core> [socket_info.c:567]: grep_sock_info(): checking if port 16060 (advertise 16060) matches port 16060
1(13464) DEBUG: <core> [socket_info.c:583]: grep_sock_info(): checking advertise if host==us: 9==15 && [8.158.178] == [192.168.158.178]
1(13464) DEBUG: usrloc [udomain.c:313]: dbrow2info(): non-local socket <8.158.178:16060>...ignoring
1(13464) CRITICAL: usrloc [udomain.c:492]: preload_udomain(): empty domain record for user some.com...skipping
1(13464) DEBUG: <core> [socket_info.c:564]: grep_sock_info(): checking if host==us: 6==15 && [58.178] == [192.168.158.178]
1(13464) DEBUG: <core> [socket_info.c:567]: grep_sock_info(): checking if port 5060 (advertise 5060) matches port 16060
1(13464) DEBUG: <core> [socket_info.c:564]: grep_sock_info(): checking if host==us: 6==15 && [58.178] == [192.168.158.178]
1(13464) DEBUG: <core> [socket_info.c:567]: grep_sock_info(): checking if port 16060 (advertise 16060) matches port 16060
1(13464) DEBUG: <core> [socket_info.c:583]: grep_sock_info(): checking advertise if host==us: 6==15 && [58.178] == [192.168.158.178]
1(13464) DEBUG: <core> [socket_info.c:564]: grep_sock_info(): checking if host==us: 6==15 && [58.178] == [192.168.158.178]
1(13464) DEBUG: <core> [socket_info.c:567]: grep_sock_info(): checking if port 5060 (advertise 5060) matches port 16060
1(13464) DEBUG: <core> [socket_info.c:564]: grep_sock_info(): checking if host==us: 6==15 && [58.178] == [192.168.158.178]
1(13464) DEBUG: <core> [socket_info.c:567]: grep_sock_info(): checking if port 16060 (advertise 16060) matches port 16060
1(13464) DEBUG: <core> [socket_info.c:583]: grep_sock_info(): checking advertise if host==us: 6==15 && [58.178] == [192.168.158.178]
1(13464) DEBUG: usrloc [udomain.c:313]: dbrow2info(): non-local socket <58.178:16060>...ignoring
1(13464) CRITICAL: usrloc [udomain.c:476]: preload_udomain(): empty username record in table location...skipping
1(13464) DEBUG: <core> [socket_info.c:564]: grep_sock_info(): checking if host==us: 2==15 && [60] == [192.168.158.178]
1(13464) DEBUG: <core> [resolve.h:355]: str2ip6(): str2ip6: ERROR: too few colons in [60]
1(13464) DEBUG: <core> [socket_info.c:583]: grep_sock_info(): checking advertise if host==us: 2==15 && [60] == [192.168.158.178]
1(13464) DEBUG: <core> [resolve.h:355]: str2ip6(): str2ip6: ERROR: too few colons in [60]
1(13464) DEBUG: <core> [socket_info.c:564]: grep_sock_info(): checking if host==us: 2==15 && [60] == [192.168.158.178]
1(13464) DEBUG: <core> [resolve.h:355]: str2ip6(): str2ip6: ERROR: too few colons in [60]
1(13464) DEBUG: <core> [socket_info.c:583]: grep_sock_info(): checking advertise if host==us: 2==15 && [60] == [192.168.158.178]
1(13464) DEBUG: <core> [resolve.h:355]: str2ip6(): str2ip6: ERROR: too few colons in [60]
1(13464) DEBUG: <core> [socket_info.c:564]: grep_sock_info(): checking if host==us: 2==15 && [60] == [192.168.158.178]
1(13464) DEBUG: <core> [resolve.h:355]: str2ip6(): str2ip6: ERROR: too few colons in [60]
1(13464) DEBUG: <core> [socket_info.c:583]: grep_sock_info(): checking advertise if host==us: 2==15 && [60] == [192.168.158.178]
1(13464) DEBUG: <core> [resolve.h:355]: str2ip6(): str2ip6: ERROR: too few colons in [60]
1(13464) DEBUG: <core> [socket_info.c:564]: grep_sock_info(): checking if host==us: 2==15 && [60] == [192.168.158.178]
1(13464) DEBUG: <core> [resolve.h:355]: str2ip6(): str2ip6: ERROR: too few colons in [60]
1(13464) DEBUG: <core> [socket_info.c:583]: grep_sock_info(): checking advertise if host==us: 2==15 && [60] == [192.168.158.178]
1(13464) DEBUG: <core> [resolve.h:355]: str2ip6(): str2ip6: ERROR: too few colons in [60]
1(13464) DEBUG: usrloc [udomain.c:313]: dbrow2info(): non-local socket <60>...ignoring
1(13464) CRITICAL: usrloc [udomain.c:492]: preload_udomain(): empty domain record for user tact...skipping
1(13464) DEBUG: <core> [socket_info.c:564]: grep_sock_info(): checking if host==us: 8==15 && [methods] == [192.168.158.178]
1(13464) DEBUG: <core> [socket_info.c:583]: grep_sock_info(): checking advertise if host==us: 8==15 && [methods] == [192.168.158.178]
1(13464) DEBUG: <core> [socket_info.c:564]: grep_sock_info(): checking if host==us: 8==15 && [methods] == [192.168.158.178]
1(13464) DEBUG: <core> [socket_info.c:583]: grep_sock_info(): checking advertise if host==us: 8==15 && [methods] == [192.168.158.178]
1(13464) DEBUG: <core> [socket_info.c:564]: grep_sock_info(): checking if host==us: 8==15 && [methods] == [192.168.158.178]
1(13464) DEBUG: <core> [socket_info.c:583]: grep_sock_info(): checking advertise if host==us: 8==15 && [methods] == [192.168.158.178]
1(13464) DEBUG: <core> [socket_info.c:564]: grep_sock_info(): checking if host==us: 8==15 && [methods] == [192.168.158.178]
1(13464) DEBUG: <core> [socket_info.c:583]: grep_sock_info(): checking advertise if host==us: 8==15 && [methods] == [192.168.158.178]
1(13464) DEBUG: usrloc [udomain.c:313]: dbrow2info(): non-local socket <methods>...ignoring
1(13464) CRITICAL: usrloc [udomain.c:476]: preload_udomain(): empty username record in table location...skipping
1(13464) CRITICAL: usrloc [udomain.c:256]: dbrow2info(): bad callid
1(13464) ERROR: usrloc [udomain.c:484]: preload_udomain(): skipping record for sip:SIPP43.some.com@192.168.158.178:5360 in table location
1(13464) DEBUG: <core> [socket_info.c:564]: grep_sock_info(): checking if host==us: 2==15 && [] == [192.168.158.178]
1(13464) DEBUG: <core> [socket_info.c:583]: grep_sock_info(): checking advertise if host==us: 2==15 && [] == [192.168.158.178]
1(13464) DEBUG: <core> [socket_info.c:564]: grep_sock_info(): checking if host==us: 2==15 && [] == [192.168.158.178]
1(13464) DEBUG: <core> [socket_info.c:583]: grep_sock_info(): checking advertise if host==us: 2==15 && [] == [192.168.158.178]
1(13464) DEBUG: <core> [socket_info.c:564]: grep_sock_info(): checking if host==us: 2==15 && [] == [192.168.158.178]
1(13464) DEBUG: <core> [socket_info.c:583]: grep_sock_info(): checking advertise if host==us: 2==15 && [] == [192.168.158.178]
1(13464) DEBUG: <core> [socket_info.c:564]: grep_sock_info(): checking if host==us: 2==15 && [] == [192.168.158.178]
1(13464) DEBUG: <core> [socket_info.c:583]: grep_sock_info(): checking advertise if host==us: 2==15 && [] == [192.168.158.178]
1(13464) DEBUG: usrloc [udomain.c:313]: dbrow2info(): non-local socket <>...ignoring
1(13464) DEBUG: <core> [socket_info.c:564]: grep_sock_info(): checking if host==us: 2==15 && [] == [192.168.158.178]
1(13464) DEBUG: <core> [socket_info.c:583]: grep_sock_info(): checking advertise if host==us: 2==15 && [] == [192.168.158.178]
1(13464) DEBUG: <core> [socket_info.c:564]: grep_sock_info(): checking if host==us: 2==15 && [] == [192.168.158.178]
1(13464) DEBUG: <core> [socket_info.c:583]: grep_sock_info(): checking advertise if host==us: 2==15 && [] == [192.168.158.178]
1(13464) DEBUG: <core> [socket_info.c:564]: grep_sock_info(): checking if host==us: 2==15 && [] == [192.168.158.178]
1(13464) DEBUG: <core> [socket_info.c:583]: grep_sock_info(): checking advertise if host==us: 2==15 && [] == [192.168.158.178]
1(13464) DEBUG: <core> [socket_info.c:564]: grep_sock_info(): checking if host==us: 2==15 && [] == [192.168.158.178]
1(13464) DEBUG: <core> [socket_info.c:583]: grep_sock_info(): checking advertise if host==us: 2==15 && [] == [192.168.158.178]
1(13464) DEBUG: usrloc [udomain.c:313]: dbrow2info(): non-local socket <>...ignoring
1(13464) CRITICAL: usrloc [udomain.c:231]: dbrow2info(): bad contact
1(13464) ERROR: usrloc [udomain.c:484]: preload_udomain(): skipping record for 8.some.com@192.168.158.178:5360 in table location
1(13464) CRITICAL: usrloc [udomain.c:256]: dbrow2info(): bad callid
1(13464) ERROR: usrloc [udomain.c:484]: preload_udomain(): skipping record for ome.com@192.168.158.178:5360 in table location
1(13464) DEBUG: <core> [socket_info.c:564]: grep_sock_info(): checking if host==us: 2==15 && [ed] == [192.168.158.178]
1(13464) DEBUG: <core> [resolve.h:355]: str2ip6(): str2ip6: ERROR: too few colons in [ed]
1(13464) DEBUG: <core> [socket_info.c:583]: grep_sock_info(): checking advertise if host==us: 2==15 && [ed] == [192.168.158.178]
1(13464) DEBUG: <core> [resolve.h:355]: str2ip6(): str2ip6: ERROR: too few colons in [ed]
1(13464) DEBUG: <core> [socket_info.c:564]: grep_sock_info(): checking if host==us: 2==15 && [ed] == [192.168.158.178]
1(13464) DEBUG: <core> [resolve.h:355]: str2ip6(): str2ip6: ERROR: too few colons in [ed]
1(13464) DEBUG: <core> [socket_info.c:583]: grep_sock_info(): checking advertise if host==us: 2==15 && [ed] == [192.168.158.178]
1(13464) DEBUG: <core> [resolve.h:355]: str2ip6(): str2ip6: ERROR: too few colons in [ed]
1(13464) DEBUG: <core> [socket_info.c:564]: grep_sock_info(): checking if host==us: 2==15 && [ed] == [192.168.158.178]
1(13464) DEBUG: <core> [resolve.h:355]: str2ip6(): str2ip6: ERROR: too few colons in [ed]
1(13464) DEBUG: <core> [socket_info.c:583]: grep_sock_info(): checking advertise if host==us: 2==15 && [ed] == [192.168.158.178]
1(13464) DEBUG: <core> [resolve.h:355]: str2ip6(): str2ip6: ERROR: too few colons in [ed]
1(13464) DEBUG: <core> [socket_info.c:564]: grep_sock_info(): checking if host==us: 2==15 && [ed] == [192.168.158.178]
1(13464) DEBUG: <core> [resolve.h:355]: str2ip6(): str2ip6: ERROR: too few colons in [ed]
1(13464) DEBUG: <core> [socket_info.c:583]: grep_sock_info(): checking advertise if host==us: 2==15 && [ed] == [192.168.158.178]
1(13464) DEBUG: <core> [resolve.h:355]: str2ip6(): str2ip6: ERROR: too few colons in [ed]
1(13464) DEBUG: usrloc [udomain.c:313]: dbrow2info(): non-local socket <ed>...ignoring
1(13464) CRITICAL: usrloc [udomain.c:256]: dbrow2info(): bad callid
1(13464) ERROR: usrloc [udomain.c:484]: preload_udomain(): skipping record for 2.168.158.178:5360 in table location
1(13464) CRITICAL: usrloc [udomain.c:231]: dbrow2info(): bad contact
1(13464) ERROR: usrloc [udomain.c:484]: preload_udomain(): skipping record for 68.158.178:5360 in table location
1(13464) ERROR: usrloc [ucontact.c:94]: new_ucontact(): no ruid for aor: 158.178:5360@e.com
1(13464) ERROR: usrloc [urecord.c:149]: mem_insert_ucontact(): failed to create new contact
1(13464) ERROR: usrloc [udomain.c:517]: preload_udomain(): inserting contact failed
1(13464) DEBUG: <core> [db_res.c:79]: db_free_columns(): freeing 21 columns

```

### Additional Information

  * **Kamailio Version** - output of `kamailio -v`

version: kamailio 4.4.7 (x86_64/linux) 35d8ab
flags: STATS: Off, USE_TCP, USE_TLS, USE_SCTP, TLS_HOOKS, DISABLE_NAGLE, USE_MCAST, DNS_IP_HACK, SHM_MEM, SHM_MMAP, PKG_MALLOC, Q_MALLOC, F_MALLOC, TLSF_MALLOC, DBG_SR_MEMORY, USE_FUTEX, FAST_LOCK-ADAPTIVE_WAIT, USE_DNS_CACHE, USE_DNS_FAILOVER, USE_NAPTR, USE_DST_BLACKLIST, HAVE_RESOLV_RES
ADAPTIVE_WAIT_LOOPS=1024, MAX_RECV_BUFFER_SIZE 262144, MAX_LISTEN 16, MAX_URI_SIZE 1024, BUF_SIZE 65535, DEFAULT PKG_SIZE 8MB
poll method support: poll, epoll_lt, epoll_et, sigio_rt, select.
id: 35d8ab 
compiled on 10:51:03 Apr 22 2018 with gcc 4.8.5

* **Operating System**:

```
CentOS 7.1
Linux linux-dev 3.10.0-514.10.2.el7.x86_64 #1 SMP Fri Mar 3 00:04:05 UTC 2017 x86_64 x86_64 x86_64 GNU/Linux
```

**Comment by miconda** [comment]
Patch related to PR #1507 has been merged and backported to 5.1. Backporting to 5.0 will be done soon.

Branch 4.4 is our of maintenance, if you want it there, make a pull request for it and I can merge it. However, there will be no new release out of branch 4.4, you have to compile from sources or build your own packages.

**Comment by tiglat** [comment]
Thank you, Daniel. I will do it myself.
